org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+{
+    "app-id": "org.siril.Siril",
```

Comment:
Can you link the Gitlab somewhere on https://siril.org/ ?

Reply:
The gitlab is linked in the FAQ: https://siril.org/faq/#how-can-i-contact-sirils-team

Reply:
thanks